src/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM python:3.8
```

Suggestion 2: as above but use the slim image
```suggestion
FROM python:3-slim
```